pyfarm/agent/service.py

```diff
@@ -118,6 +115,60 @@ def agents_endpoint(cls):
         """
         return config["master_api"] + "/agents/"
 
+    @inlineCallbacks
+    def schedule_call(self, delay, function, *args, **kwargs):
+        """
+        Schedules ``function`` to be run after ``delay`` has elapsed.
```

Comment:
I'll rename this then to something like 'repeating_call' or something similar.  I carried over the old name initially but you're right that it does not make as much sense given the context.